README.md

```diff
@@ -148,6 +148,7 @@ both for the command line and for other platforms:
     - [DistroWatch](https://distrowatch.com/dwres.php?resource=man-pages)
     - [tldr.ooops.me](https://tldr.ooops.me): web client with multilingual support
     - [TLDR Persian](https://opoet7.github.io/tldr-persian/): Web Client in Persian
+- [Zig client](https://gitlab.com/ve-nt/outfieldr)
```

Comment:
You choose a cool name for your client, I think you should use it
```suggestion
    - [Outfieldr](https://gitlab.com/ve-nt/outfieldr): A Zig client
```

Reply:
Thanks! I agree with this, although I don't think the whitespace before the checkmark is intentional?

Reply:
No that white space is not intentional ;)